src/ws/hooks/item.test.ts

```diff
@@ -44,17 +43,12 @@ describe('Ws Item Hooks', () => {
       const itemEvent = {
         kind: KINDS.SELF,
         op: OPS.UPDATE,
-        item: JSON.parse(JSON.stringify(newItemRecord.toJS())),
+        item: JSON.parse(JSON.stringify(newItem)),
```

Comment:
This is needed because we parse the date into a `Date`, if we keep the serialized version we won't need this ...